src/Apps/CheezeWizards/Screens/WizardScreen.js

```diff
@@ -67,7 +67,7 @@ export default class WizardScreen extends React.Component {
 
   async componentDidMount() {
     try {
-      const {id, affinity, ascending, ascensionOpponent, currentDuel, maxPower, molded, nonce, power, ready} = this.props.navigation.state.params.wizard;
```

Comment:
This logic confused me @markspereira - I have followed the pattern to get it working but I think if this is to remove the additional values which are indexed based, we go do this in the service class and save the inline code here, meaning a wizard is always of a given structure.

Reply:
Yep that's exactly what I tried to do. Let's move it into the service class that would be a lot better